django/api/migrations/0008_chargerrebates.py

```diff
@@ -1,4 +1,5 @@
-# Generated by Django 3.1.6 on 2021-12-06 17:12
+# Generated by Django 3.1.6 on 2021-12-13 20:49
```

Comment:
Please generate a new migration file for the changes to the column types. This can cause issues to our existing environments that have already mgirated

Reply:
I may have removed the wrong line while fixing the merge conflict.